src/utils/tree.js

```diff
@@ -1,28 +1,22 @@
-import FOLDERS from "./folders.json";
-
-const USE_MOCK_DATA = true;
-
-export function buildTree(level, folderTree) {
-  if (USE_MOCK_DATA) {
-    if (!level) {
-      folderTree.push(FOLDERS[0].id, FOLDERS[0].title, []);
-    }
-
-    const parentId = folderTree[0];
-    const temp = folderTree;
-    for (let i = 1; i < FOLDERS.length; i++) {
-      if (FOLDERS[i]["parent_folder"] === parentId) {
-        temp.push([FOLDERS[i].id, FOLDERS[i].title, FOLDERS[i].bookmark]);
-      }
-    }
+export function buildTree(folderList, level, folderTree) {
+  if (!level) {
+    folderTree.push(folderList[0].id, folderList[0].title, []);
+  }
 
-    for (let i = 3; i < temp.length; i++) {
-      const subTree = temp[i];
-      buildTree(level + 1, subTree);
+  const parentId = folderTree[0];
+  const temp = folderTree;
+  for (let i = 1; i < folderList.length; i++) {
```

Comment:
```suggestion

  for (let i = 1; i < folderList.length; i++) {
```
for문과 변수 할당 사이에 개행해주시면 좋을 것 같습니다!

Reply:
개행완료했습니다! 감사합니다~